index.js

```diff
@@ -177,11 +184,11 @@ function joinPaths(inputPathOrPaths, outputPaths) {
   }
 }
 
-async function analyzeFiles(cwd, files, options) {
+function analyzeFiles(cwd, files, options) {
   let allTranslationKeys = new Map();
 
   for (let file of files) {
-    let translationKeys = await analyzeFile(cwd, file, options);
+    let translationKeys = analyzeFile(cwd, file, options);
```

Comment:
looks like there are still plenty of places where async/await was removed

Reply:
@Turbo87 those don't actually return promises, they are sync functions.
So basically for some reason sync functions had `async` keyword which I would say is also confusing